docs/Nutrition1.md

```diff
@@ -0,0 +1,58 @@
+# Meal Planner Documentation
+
+## Overview
+
+## Step 1
+#Meal plan log in page
+
+- Admin shoud be able to access the log in page using the assigned user name and password after inputting the web address [https://goodmealplan.com/](https://goodmealplan.com/) .
+
+![Log in_page-0001](https://imgur.com/vjoJciS.jpg)
```

Comment:
Please checkin images alongside the .md files instead of using 3rd party sites.

Reply:
Corrections effected.